components/Main/Profile.js

```diff
@@ -0,0 +1,27 @@
+const Profile = `
```

Comment:
Hi @ogeobubu, make your modules functions that return the HTML code. This is because, moving forward, we might need to give them arguments/parameters. I explained more of this in the course. I'll recommend you watch it.